msteams-platform/m365-apps/prerequisites.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Set up your dev environment for extending Teams apps across Microsoft 365
+description: Here are the prerequisites for extending your Teams apps across Microsoft 365
+ms.date: 11/15/2021
+ms.topic: how-to
+---
+# Set up your dev environment for extending Teams apps across Microsoft 365 (preview)
+
+The development environment for extending Teams apps across Microsoft 365 is similar to what you use for Microsoft Teams development. This article discusses the configurations required for your test tenant to run preview builds of Microsoft Teams and Microsoft Office applications. These changes are needed because this feature is in preview.
+
+## Overview
+> [!div class="checklist"]
+> * Obtain an M365 Developer (Sandbox) Tenant and enable sideloading
+> * Enroll your M365 tenant in *Office 365 Targeted Releases*
+> * Configure your account to access preview versions of Outlook and Office apps
+> * Install Teams Toolkit vscode extension (optional)
```

Comment:
Suggestion, provide links to the respective topics.

Reply:
Done in next commit